### PR TITLE
WIKI-1348 Restore uncache page containing selected macros capability

### PR DESCRIPTION
**Pull request by boubaker**
This PR Reverts changes made that disables the capability to uncache pages containing some selected macros, which was able to be made using Component Plugins:
https://docs.exoplatform.org/public/index.jsp?topic=/PLF50/PLFRefGuide.Configurations.ExternalComponentPlugins.Wiki.WikiUncache.html

The Plugin can be added on WikiService instead of PageRenderingCacheService that was deleted.
A doc issue will be opened to update documentation.

### CODE REVIEW

File: wiki-service/src/main/java/org/exoplatform/wiki/service/WikiService.java [hunk 1]
```diff
@@ -771,4 +772,11 @@
    */
   public int getUploadLimit();
 
+  /**
+   * Uncache page rendering fro pages containing a chosen macro
```
Comment (thomasdelhomenie) [inline wiki-service/src/main/java/org/exoplatform/wiki/service/WikiService.java]: Typo : "fro" -> "for"

File: wiki-service/src/main/java/org/exoplatform/wiki/service/impl/WikiServiceImpl.java [hunk 2]
```diff
@@ -550,6 +551,12 @@ public Page getPageOfWikiByName(String wikiType, String wikiOwner, String pageNa
     return page;
   }
 
+  @Override
+  public void addUnCachedMacro(UnCachedMacroPlugin plugin) {
+    for (String name : plugin.getUncachedMacroes())
+      uncachedMacroes.add(name);
```
Comment (thomasdelhomenie) [inline wiki-service/src/main/java/org/exoplatform/wiki/service/impl/WikiServiceImpl.java]: Could be replaced by: 

uncachedMacroes.addAll(plugin.getUncachedMacroes());

File: wiki-service/src/main/resources/conf/portal/cache-configuration.xml [hunk 3]
```diff
@@ -105,9 +105,9 @@
     </component-plugin>
   </external-component-plugins>
 
-  <!--
+  <!--  -->
```
Comment (thomasdelhomenie) [inline wiki-service/src/main/resources/conf/portal/cache-configuration.xml]: Can be removed

File: wiki-service/src/main/java/org/exoplatform/wiki/service/WikiService.java (hunk 1 above)
Comment (boubaker) [reply wiki-service/src/main/java/org/exoplatform/wiki/service/WikiService.java]: done

File: wiki-service/src/main/java/org/exoplatform/wiki/service/impl/WikiServiceImpl.java (hunk 2 above)
Comment (boubaker) [reply wiki-service/src/main/java/org/exoplatform/wiki/service/impl/WikiServiceImpl.java]: done

File: wiki-service/src/main/resources/conf/portal/cache-configuration.xml (hunk 3 above)
Comment (boubaker) [reply wiki-service/src/main/resources/conf/portal/cache-configuration.xml]: done

**Pull request merged**